config/locales/en.yml

```diff
@@ -1,230 +1,5 @@
-        one: is too short (minimum is 1 character)
-        other: Please make your password stronger. At least 6 characters.
-      wrong_length:
-        one: is the wrong length (should be 1 character)
-        other: is the wrong length (should be %{count} characters)
-      other_than: must be other than %{count}
-    template:
-      body: 'There were problems with the following fields:'
-      header:
-        one: 1 error prohibited this %{model} from being saved
-        other: "%{count} errors prohibited this %{model} from being saved"
-    company_request:
-      email:
-        format: "Sorry, that doesn't appear to be a valid email address. Please enter your email address to request a company profile."
-        presence: "Please enter your email so we can notify you when we have created the profile."
-  helpers:
-    select:
-      prompt: Please select
-    submit:
-      create: Create %{model}
-      submit: Save %{model}
-      update: Update %{model}
-  number:
-    currency:
-      format:
-        delimiter: ","
-        format: "%u%n"
-        precision: 2
-        separator: "."
-        significant: false
-        strip_insignificant_zeros: false
-        unit: "$"
-    format:
-      delimiter: ","
-      precision: 3
-      separator: "."
-      significant: false
-      strip_insignificant_zeros: false
-    human:
-      decimal_units:
-        format: "%n %u"
-        units:
-          billion: b
-          million: m
-          quadrillion: q
-          thousand: k
-          trillion: t
-          unit: ''
-      format:
-        delimiter: ''
-        precision: 3
-        significant: true
-        strip_insignificant_zeros: true
-      storage_units:
-        format: "%n %u"
-        units:
-          byte:
-            one: Byte
-            other: Bytes
-          gb: GB
-          kb: KB
-          mb: MB
-          tb: TB
-    percentage:
-      format:
-        delimiter: ''
-        format: "%n%"
-    precision:
-      format:
-        delimiter: ''
-  support:
-    array:
-      last_word_connector: ", and "
-      two_words_connector: " and "
-      words_connector: ", "
-  time:
-    am: am
-    formats:
-      w3c: '%Y-%m-%dT%H:%M:%S+00:00'
-      default: "%a, %d %b %Y %I:%M:%S %p %Z"
-      long: "%B %d, %Y %I:%M %p"
-      short: "%d %b %I:%M %p"
-      table: "%d %b %Y"
-      file: '%Y-%m-%d-%H-%M'
-      post: '%e %B %Y'
-    pm: pm
-  devise:
-    sessions:
-      signed_out: ''
-  activerecord:
-    errors:
-      models:
-        user:
-          attributes:
-            password_confirmation:
-        company:
-          attributes:
-            ticker:
-              taken: "%{value} already registered for selected stock market."
-  filter:
-    company_name_cont: Filter by company name
-    acquired_company_name_cont: Filter by acquired company name
+      invalid_email_format: "The email format is invalid."
+      email_not_found: "The email address does not exist in our records."
```

Comment:
The new error messages added to the locale file should be more descriptive to help users understand the issue better. Consider providing additional context or suggestions for resolving the error.